python/lsst/afw/display/scaling.cc

```diff
@@ -0,0 +1,247 @@
+/*
```

Comment:
Please add the LSST legal boilerplate, or the appropriate copyright if different